.github/workflows/docs-workflow.yaml

```diff
@@ -1,3 +1,23 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2023 SECO Mind S.r.l.
```

Comment:
We are writing everywhere `SECO Mind Srl`, let's do the same for consistency reasons.